tsl/src/fdw/data_node_scan_plan.c

```diff
@@ -297,33 +297,15 @@ ts_ec_member_matches_foreign(PlannerInfo *root, RelOptInfo *rel, EquivalenceClas
 	return true;
 }
 
-static void
-add_data_node_scan_paths(PlannerInfo *root, RelOptInfo *baserel)
+/*
+ * Build parameterizations that are useful for performing joins with the given
+ * hypertable relation. We will use them to generate the parameterized data node
+ * scan paths. The code is mostly copied from postgres_fdw,
+ * postgresGetForeignPaths().
+ */
+static List *
+build_parameterizations(PlannerInfo *root, RelOptInfo *hyper_rel)
```

Comment:
This is now a separate function, because it makes no sense to call it for each data node, it does the same work.